adobe/spectrum-web-components
# fix(picker,action-menu,split- button): update interaction model

## Description
Support opening the element's Menu and selecting one of its Menu Items with a single pointer button interaction.

## Related issue(s)
- fixes #2966

## How has this been tested?
    1. Go [here](https://picker-interaction--spectrum-web-components.netlify.app/storybook/?path=/story/menu-submenu--submenu)
    2. Make sure the Action Menu is closed
    3. Pointerdown on the Action Button
    4. See that the Menu opens
    5. With the pointer STILL DOWN move to a Menu Item in the _second_ fly out Menu
    6. Release the pointer
    7. See that all of the correct "selections" were made
    1. Go [here](https://picker-interaction--spectrum-web-components.netlify.app/storybook/?path=/story/picker--default)
    2. Pointerdown on the Picker
    3. Keep it down
    4. See that the Menu opened
    5. Move to a Menu Item
    6. Release the pointer
    7. See that the selection is made
-   [ ] _Test case 3_ For good measure
    1. Go [here](https://picker-interaction--spectrum-web-components.netlify.app/storybook/?path=/story/picker--default)
    2. Click on the Picker
    3. See that the Menu opened
    4. Move to a Menu Item
    5. Click the Menu Item
    6. See that the selection is made

## Types of changes

## Checklist
